tests/test_v1_api.py

```diff
@@ -95,6 +95,15 @@ def test_comparison_report_api(self, mock_fetch_systems):
             headers=fixtures.AUTH_HEADER,
         )
         self.assertEqual(response.status_code, 200)
+        comparisons = json.loads(response.data)
+        # we hard-code the index lookup since we know the fixture layout
+        network_comparisons = comparisons["facts"][12]["comparisons"]
+        ipv4_comparison = [
+            comparison
+            for comparison in network_comparisons
+            if comparison["name"] == "eth99.ipv4_addresses"
+        ][0]
+        self.assertEqual(ipv4_comparison["state"], "DIFFERENT")
```

Comment:
Could we add individual tests for each unique fact to ensure none of them regress?